wireless_modules/middle_module/test.py

```diff
@@ -0,0 +1,51 @@
+# Author: Kunj
+# Last Modified: 3/08/2020
+import paho.mqtt.client as mqtt
+import time
+
+client = mqtt.Client("my_id")
+
+MODULE_NUM = "2"
+START_TOPIC = '/v3/wireless-module/{}/start'.format(MODULE_NUM)
```

Comment:
Try [f-strings](https://realpython.com/python-f-strings/#simple-syntax) for cleaner syntax

Reply:
Oh damn never knew this exited 😂